workloads/router-perf-v2/common.sh

```diff
@@ -183,7 +185,8 @@ gen_mb_mix_config(){
       local scheme=https
       local port=443
     fi
-    while read host; do
+    oc get route -n http-scale-${termination} --no-headers | awk '{print $2}' > /tmp/temp-route-mb-mix.txt
```

Comment:
there is no such namespace as http-scale-mix. mix will get routes from  http edge passthrough reencrypt namespaces

Reply:
Got it Mohit. I somehow copied the oc get route command in `gen_mb_mix_config()` incorrectly which led to me writing `http-scale-${termination}` instead of `http-scale-${mix_termination} `